File: arrangeCoins.py
# https://leetcode.com/problems/arranging-coins/description/

class Solution(object):
	def arrangeCoins(self, n):
		k = 0
		while n > 0:
			k += 1
			n -= k

		if n < 0: 
			k -= 1

		return k

File: arrangeCoins.java
# https://leetcode.com/problems/arranging-coins/description/

class Solution {
    public int arrangeCoins(int n) {
        int k = 0;
        while (n - k > 0) {
            k++;
            n -= k;
        }
        return k;        
    }
}
